### PR TITLE
add edison-node.js for Intel Edison board

### PR DESCRIPTION
**Pull request by ZhijunZhao**
@zikalino

### CODE REVIEW

File: edison-node.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+  gulp.task('init', 'Initializes sample', function (cb) {
+
+    if (options.configPostfix && options.configTemplate) {
+      all.updateGlobalConfig(options.configPostfix, options.configTemplate);
+    }
+
+    cb();
+  })
+
```
Comment (ZhijunZhao) [inline edison-node.js:34]: On newly flashed device, the node version is 4.4. So I think we don't need 'install-tools' task. 

Will consider merging this js with 'raspberrypi-node' file later.

File: edison-node.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    all.uploadFilesViaScp(filesLocal, filesRemote, function () {
+      console.log("- Installing NPM packages on the device. It might take several minutes.");
+      all.sshExecCmd('cd ' + targetFolder + ' && npm install', { verbose: args.verbose }, cb);
+    });
+  });
+
+  gulp.task('run-internal', false, function (cb) {
+    var targetFolder = config.project_folder ? config.project_folder : '.';
+    var startFile = config.start_file ? config.start_file : 'app.js';
+    var nodeCommand = 'node';
```
Comment (ZhijunZhao) [inline edison-node.js:55]: The command is 'node' instead of 'nodejs'

File: edison-node.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
+/*
+* Gulp Common - Microsoft Sample Code - Copyright (c) 2016 - Licensed MIT
+*/
+'use strict';
+
+var args = require('get-gulp-args')();
+var fs = require('fs');
+
+/**
+ * Main entry point for all Rapberry Pi Node configuration.
```
Comment (xscript) [inline edison-node.js]: "Raspberry Pi Node"?

File: edison-node.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+ * @param {object} gulp     - Gulp instance
+ * @param {object} options  - Raspberry Pi Node Specific options
```
Comment (xscript) [inline edison-node.js]: same here

**Review by xscript — APPROVED** [review]

**Pull request merged**